src/Test/LocalTest.java

```diff
@@ -0,0 +1,8 @@
+package Test;
+
+public class LocalTest {
+    public static void main(String[] args) {
+        System.out.println();
```

Comment:
I guess we should add verification for local test!!

Reply:
upgraded. check pls

Reply:
looks nice thanks